networks/_modules/docker-besu/tessera.tf

```diff
@@ -50,8 +50,19 @@ resource "docker_container" "tessera" {
     <<EOF
 echo "Tessera${count.index + 1}"
 
-START_TESSERA="java -Xms128M -Xmx128M \
-  -jar ${lookup(var.tessera_app_container_path, count.index, "/tessera/tessera-app.jar")} \
+// TODO: remove following block when Jigsaw dist is default
+FILE="${lookup(var.tessera_app_container_path, count.index, "/tessera/tessera-app.jar")}"
+
+if [[ -f "$${FILE}" ]];
+then
+  COMMAND="java -Xms128M -Xmx128M -jar $${FILE}"
+else
+  JAVA_OPTS="-Xms128M -Xmx128M"
+  COMMAND="/home/tessera-extracted/bin/tessera"
```

Comment:
this should be above the `// TODO: remove following block when Jigsaw dist is default` comment, as this needs to be kept in the future. then we can **only** remove the block as it says (and we keep a variable for command which makes things more clear).

same in the other `tessera.tf`.

Reply:
Good idea. I made the change, please check it is what you were thinking of

Reply:
that looks good